db/migrate/20210810093230_add_unarchived_at_to_documents.rb

```diff
@@ -0,0 +1,5 @@
+class AddUnarchivedAtToDocuments < ActiveRecord::Migration[6.1]
+  def change
+    add_column :documents, :unarchived_at, :datetime
```

Comment:
I think you meant to remove this?

Reply:
I removed the `archived` boolean but I thought this datestamp was probably still useful?

Reply:
Ah, missed that you're still setting it. I don't know why it would be useful? I'm resistant to adding things we don't explicitly need.